### PR TITLE
Create jacobschoen2.md

### PR DESCRIPTION
**Pull request by jkschoen**

### CODE REVIEW

**Comment by derekoverby** [comment]
what the hell is this crap?

**Comment by summittr** [comment]
:-1:

**Comment by summittr** [comment]
:-1: :-1: :-1:

**Comment by jkschoen** [comment]
nice to see we are friendly here

**Comment by summittr** [comment]
lol

File: jacobschoen2.md [hunk 1]
```diff
@@ -0,0 +1 @@
+some content
```
Comment (derekoverby) [inline jacobschoen2.md]: im pretty sure you mis-spelled "crap"

**Comment by derekoverby** [comment]
@isayyad can you help rob with git?

**Comment by derekoverby** [comment]
:8ball:

**Comment by derekoverby** [comment]
if you merge this crap i quit

**Comment by derekoverby** [comment]
(╯°□°）╯︵ ┻━┻

**Pull request merged**